engines/ultima/shared/core/str.h

```diff
@@ -45,6 +45,7 @@ class String : public Common::String {
 	String(const String &str) : Common::String(str) {}
 	String(const Common::String &str) : Common::String(str) {}
 	explicit String(char c) : Common::String(c) {}
+	using Common::String::operator=;
```

Comment:
Unless I'm wrong, the using keyword is a C++11 keyword. The engine doesn't have any C++11 required yet, so I don't want to have to add in the requirement just for this.

Reply:
On this context it's not C++11. The C++11 form is a typedef replacement.